### DISCUSSION TITLE
French translation

### DISCUSSION THREAD

**Discussion by edimitchel** [vuejs-translations/guidelines#2] [Q&A] [answered]
As we all know that French is the most loved language (after english), isn't @yyx990803? 😀

**Comment by edimitchel** [comment]
Please subscribe to that discussion to get notified!

**Comment by edimitchel** [comment, marked as answer]
I create a project to start translation.
I'm working for organizing tasks.
https://github.com/vuejs-translations/docs-fr

**Reply by JonJaggerJack** [reply]
Hi !
I'd like to help

**Reply by edimitchel** [reply]
Please, let take a look into issues and peek one page to translate, fork and open PR :)

**Reply by edimitchel** [reply]
FYI, the repo has been [transferred to the vuejs-translation organization](https://github.com/vuejs-translations/docs-fr)!

**Comment by LaurentSauzetOff** [comment]
I wanna help too ! Can you explain to me how I should do this ?

**Reply by edimitchel** [reply]
Let fork my repo, and choose a page to translate :)

**Comment by JonJaggerJack** [comment]
I want to help too

De : Laurent ***@***.***>
Envoyé le :mardi 19 juillet 2022 17:35
À : ***@***.***>
Cc : Jonas ***@***.***>; ***@***.***>
Objet :Re: [vuejs-translations/guidelines] French translation (Discussion #2)


I wanna help too ! Can you explain to me how I should do this ?

**Reply by edimitchel** [reply]
Let fork my repo, and choose a page to translate :)

**Comment by kissu** [comment]
Do we still need some baguette 🥖 translations?  😁 💛 
Good 🇫🇷 speaker with decent grammar here. 🖐🏻

**Reply by edimitchel** [reply]
Join us on my repo :)

**Comment by JonJaggerJack** [comment]
Hi !!
I'd like to help with the translation but until now I can not access the project and the job.
Can u guide me?
________________________________
